temporalcli/commandsgen/commands.yml

```diff
@@ -161,10 +161,14 @@ commands:
           Default is "info" for most commands and "warn" for `server start-dev`.
         default: info
       - name: log-format
-        type: string
+        type: string-enum
         description: |
           Log format.
-          Options are: text, json.
+          "pretty" is an alias for "text". This is for compatibility reasons, please use "text".
+        enum-values:
+          - text
+          - json
+          - pretty
```

Comment:
@cretz Was there some reason we wanted this not to be a string-enum? I remember you avoided it specifically for some reason but I think maybe that didn't get documented, so I'm not sure if it's relevant anymore.

Reply:
I think it is because we wanted "pretty" to be an _undocumented_ enumerate alias. I think #678 could help here.